Broadcom/symantec-endpoint-protection/_meta/manifest.yml

```
@@ -1,6 +1,6 @@
 uuid: 419bd705-fa61-496c-94fa-28d6c1f2e2a8
-name: Symantec Endpoint Protection
-slug: symantec-endpoint-protection
+name: Symantec/Broadcom Endpoint Security
+slug: symantec-broadcom-endpoint-security
```

I prefer `<vendor>-<product name>`
```suggestion
slug: broadcom-symantec-endpoint-security
```